### PR TITLE
release: update 1.5 version download path.

### PR DESCRIPTION
**Pull request by shuaijinchao**

### CODE REVIEW

File: content/downloads/index.zh.md [hunk 1]
```diff
@@ -101,6 +101,18 @@ include_footer: true
           )
         </div>
       </div>
+      <div class="flex-table row" role="rowgroup">
+        <div class="flex-row first" role="cell"><span class="flag-icon flag-icon-gb"></span>1.5</div>
+        <div class="flex-row" role="cell">2020 August 5</div>
+        <div class="flex-row" role="cell">
+          <a href="https://downloads.apache.org/apisix/1.5/">source</a>
```
Comment (juzhiyuan) [inline content/downloads/index.zh.md]: Please use `源码` instead

File: content/downloads/index.zh.md [hunk 2]
```diff
@@ -101,6 +101,18 @@ include_footer: true
           )
         </div>
       </div>
+      <div class="flex-table row" role="rowgroup">
+        <div class="flex-row first" role="cell"><span class="flag-icon flag-icon-gb"></span>1.5</div>
+        <div class="flex-row" role="cell">2020 August 5</div>
```
Comment (juzhiyuan) [inline content/downloads/index.zh.md]: 2020-08-05

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

**Comment by shuaijinchao** [comment]
@juzhiyuan please recheck.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Pull request merged**